src/types/MeldekortTypes.ts

```diff
@@ -1,28 +1,46 @@
+import {Periode} from "./Periode";
+
 export type MeldekortListe = {
   meldekort: Meldekort[];
 };
 
+export type MeldekortUtenDager = {
+    id: string;
+    fom: Date;
+    tom: Date;
+    status: string;
+};
+
 export type Meldekort = {
-  id: string;
-  fom: Date;
-  tom: Date;
-  antallDagerPåTiltaket: number;
-  tiltaksType: string;
-  meldekortUke1: MeldekortDag[];
-  meldekortUke2: MeldekortDag[];
+    id: string;
+    fom: Date;
+    tom: Date;
+    antallDagerPåTiltaket: number,
+    tiltak: Tiltak[];
+    meldekortDager: MeldekortDag[];
 };
 
 export type MeldekortDag = {
-  dag: string;
-  dato: Date;
-  status: MeldekortStatus;
+    dato: Date;
+    dag: string;
+    tiltak: Tiltak;
+    status: MeldekortStatus;
 };
 
+export type Tiltak = {
+    id: string;
+    periode: Periode;
+    typeBeskrivelse: string;
+    typeKode: string;
+    antDagerIUken: number;
```

Comment:
Supert om denne variabelen kan hete ‘antallDagerIUken’ isteden. For lengre oppe er det en variabel som heter ‘antallDagerPåTiltaket’. Så det hadde vært knall om de kunne ha samme format :D